### PR TITLE
Readding AWS Warehouse world to build in Rolling / Jammy

### PR DESCRIPTION
**Pull request by SteveMacenski**
Per https://github.com/aws-robotics/aws-robomaker-small-warehouse-world/issues/23, I'm readding AWS Warehouse world to build on Rolling now that Gazebo dependency is met to trigger release jobs. Was disabled in https://github.com/ros/rosdistro/pull/32036/files#diff-a79d50d14101a47dfc4ea7b0dd7b9446f34b544807d8f0c48cc6a8f07fc3b36aL357.

### CODE REVIEW

File: rolling/distribution.yaml [hunk 1]
```diff
@@ -5644,7 +5645,6 @@ repositories:
       tags:
         release: release/rolling/{package}/{version}
       url: https://github.com/ros2-gbp/webots_ros2-release.git
-      version: 1.2.3-1
```
Comment (nuclearsandwich) [inline rolling/distribution.yaml:5647]: This looks like a stray edit. If not, why was it removed?

File: rolling/distribution.yaml [hunk 2]
```diff
@@ -404,6 +404,7 @@ repositories:
       tags:
         release: release/rolling/{package}/{version}
       url: https://github.com/aws-gbp/aws_robomaker_small_warehouse_world-release.git
+      version: 1.0.5-1
```
Comment (nuclearsandwich) [inline rolling/distribution.yaml:407]: Bloom artifacts are specific to a target distribution and the release version being re-added here does not contain artifacts for Ubuntu Jammy.

File: rolling/distribution.yaml [hunk 3]
```diff
@@ -5625,7 +5626,7 @@ repositories:
     doc:
       type: git
       url: https://github.com/cyberbotics/webots_ros2.git
-      version: master
+      version: rolling
```
Comment (nuclearsandwich) [inline rolling/distribution.yaml:5629]: Stray edit? If not, the changes to webots_ros2 probably belong in a separate PR.

**Review by nuclearsandwich — CHANGES_REQUESTED** [review]
There appear to be a few changes in an unrelated package. Those changes should either be dropped entirely if they were made by mistake or removed to a separate PR.

Bloom artifacts are specific to a target distribution and the release version being re-added here does not contain artifacts for Ubuntu Jammy.

**Comment by SteveMacenski** [comment]
Oh weird - I just edited in the browser I have no idea why those edits were made. 

Closing from discussion in https://github.com/aws-robotics/aws-robomaker-small-warehouse-world/issues/23#issuecomment-1146297872
